### PR TITLE
BaSP-TG-161: Table and searchbar styles

### PR DESCRIPTION
**Pull request by renzopimentel**
Changed the style in tables, modifying the colors of the fonts and actions. 
Modify the add button and the searchbar with the icon

### CODE REVIEW

File: src/Components/Shared/Table/Table.module.css [hunk 1]
```diff
@@ -24,6 +30,47 @@
     margin: auto;
 }
 
+.searchWrap{
+    float: left;
+	width: 250px;
+}
```
Comment (CristianBucci) [inline src/Components/Shared/Table/Table.module.css:36]: Fix identation!

File: src/Components/Shared/Table/Table.module.css [hunk 2]
```diff
@@ -24,6 +30,47 @@
...
+
+.searchWrap .searchBox{
+	position: relative;
+	width: 200px;
+	height: 40px;
```
Comment (CristianBucci) [inline src/Components/Shared/Table/Table.module.css:41]: Same as above

File: src/Components/Shared/Table/Table.module.css [hunk 3]
```diff
@@ -24,6 +30,47 @@
...
+.searchWrap .searchBox .input{
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	padding: 10px 20px;
+	border-radius: 3px;
+	font-size: 18px;
+}
```
Comment (CristianBucci) [inline src/Components/Shared/Table/Table.module.css:53]: Same as above!

File: src/Components/Shared/Table/Table.module.css [hunk 4]
```diff
@@ -24,6 +30,47 @@
...
+	position: absolute;
+	top: 0;
+	right: 0;
+	width: 60px;
+	height: 100%;
+	z-index: 0;
+	cursor: pointer;
+    border: none;
+    background-color: none;
+    border-radius: 0 15px 15px 0;
```
Comment (CristianBucci) [inline src/Components/Shared/Table/Table.module.css:65]: Same as above!

**Review by matescarabino — APPROVED** [review]

**Review by Ansitox — APPROVED** [review]

**Pull request merged**